src/styles/card.scss

```diff
@@ -0,0 +1,66 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__info {
+    font-family: $font;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $mainColor;
+    padding-bottom: 4px;
+  }
+
+  &__productCode {
+    font-family: $font;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: $fontWeightStandard;
+    padding-bottom: 16px;
+    color: $secondColor;
+  }
+
+  &__buyButton {
+    height: 40px;
+    width: 100%;
+    background-image: url(../images/button.png);
```

Comment:
Why do you use background for button? It's redundant
```suggestion
```

Reply:
don't understand at all
I have to have an image for button

Reply:
> don't understand at all I have to have an image for button

use background-color. no need to import button background as png image.